.mocharc.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
     'docs/.next/**',
   ],
   recursive: true,
+  timeout: (process.env.CIRCLECI === 'true' ? 3 : 1) * 1000, // Circle CI has low-performance CPUs.
```

Comment:
```suggestion
  timeout: (process.env.CIRCLECI === 'true' ? 3 : 2) * 1000, // Circle CI has low-performance CPUs.
```

Default timeout is 2s as far as I can tell:

> --timeout, -t, --timeouts  Specify test timeout threshold (in milliseconds)
                                                        [string] [default: 2000]

-- https://mochajs.org/#command-line-usage

Reply:
I was proposing 1s instead of 2s to make it easier (sooner) to catch slow tests during the development phase. What do you think about it?

Reply:
> I was proposing 1s instead of 2s to make it easier (sooner) to catch slow tests during the development phase. What do you think about it?

I don't think that's a good idea. It slows down local development with a potential high false positive rate. It also doesn't target the problem description of this PR.

Reply:
> with a potential high false positive rate

Yeah, this is the part I wonder the most about. What would be a good threshold for our test? It seems that 2s is a lot more than enough.

Reply:
> What would be a good threshold for our test? 

Just keep the default value.